.travis.yml

```diff
@@ -42,20 +42,22 @@ before_script:
   - wstool merge $CI_SOURCE_PATH/mbzirc.rosinstall
   - wstool up
 
-  # package depdencies: install using rosdep.
-  - cd ~/catkin_ws
-  - rosdep install -y -q -r --from-paths src --ignore-src --rosdistro $ROS_DISTRO
-
 # Compile and test.
 script:
   - source /opt/ros/$ROS_DISTRO/setup.bash
+  # package depdencies: install using rosdep.
   - cd ~/catkin_ws
-  - catkin build -p1 -j1
-  - catkin run_tests -p1 -j1
-  - catkin_test_results --all build
+  - rosdep install -y -q -r -n --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+  # build packages
+  - catkin build -p1 -j1 --no-status --summarize
+  - source ~/catkin_ws/devel/setup.bash
+  # run tests
+  - xvfb-run -a --server-args="-screen 0 800x600x24" catkin build --catkin-make-args run_tests -- -p1 -j1 --limit-status-rate 0.017 --summarize $REPOSITORY_NAME --no-deps
```

Comment:
@tongtygj Right, now gazebo camera plugin doesn't work if rendering is disabled, and Travis server has no screen to render. Instead by using Xvfb, we can use virtual screen to render.

-- ⌘ Yuki Furuta

2016年5月2日 12:38 +0900 に 趙　漠居<notifications@github.com>は書きました:
>  
> In.travis.yml(https://github.com/start-jsk/jsk_mbzirc/pull/70#discussion_r61700308):
>  
> >- cd ~/catkin_ws>- - catkin build -p1 -j1>- - catkin run_tests -p1 -j1>- - catkin_test_results --all build>+ - rosdep install -y -q -r -n --from-paths src --ignore-src --rosdistro $ROS_DISTRO>+ # build packages>+ - catkin build -p1 -j1 --no-status --summarize>+ - source ~/catkin_ws/devel/setup.bash>+ # run tests>+ - xvfb-run -a --server-args="-screen 0 800x600x24" catkin build --catkin-make-args run_tests -- -p1 -j1 --limit-status-rate 0.017 --summarize $REPOSITORY_NAME --no-deps
>  
> @furushchev(https://github.com/furushchev)
> Just want to learn somthing... So "xvfb-run" is the solution for the problem of display output about task2(husky)?
>  
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly orview it on GitHub(https://github.com/start-jsk/jsk_mbzirc/pull/70/files/d8fae126083887e5c0ea0b3a2d079988df2b17ff#r61700308)
>  
>  
>